<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCPHA</class>
 <widget class="QMainWindow" name="MCPHA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multichannel Pulse Height Analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="controlWidget" native="true">
      <layout class="QHBoxLayout" name="controlLayout">
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addrValue">
         <property name="text">
          <string>192.168.1.100</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="syncCheck">
         <property name="text">
          <string>synchronous histograms</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="neg1Check">
         <property name="text">
          <string>negative CH1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="neg2Check">
         <property name="text">
          <string>negative CH2</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="rateLabel">
         <property name="text">
          <string>decimation factor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rateValue"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
